### PR TITLE
Fix image loader in FF

### PR DESCRIPTION
**Pull request by Pessimistress**
For #317 

### Change List

- `ImageLoader` loads to HTML object if possible (best compatibility)
- detect `createImageBitmap` support for options

### CODE REVIEW

File: modules/images/src/image-loader.js [hunk 1]
```diff
@@ -13,7 +8,7 @@ export default {
   name: 'Images',
   extensions: EXTENSIONS,
   parse: canParseImage && parseImage,
-  loadAndParse: !canParseImage && loadImage
+  loadAndParse: typeof Image !== 'undefined' && loadToHTMLImage
```
Comment (ibgreen) [inline modules/images/src/image-loader.js]: Why this change? This no longer captures whether the node image parser has been installed?

Is `canParseImage` used at all after this change?

Comment (Pessimistress) [reply modules/images/src/image-loader.js]: It's still used when in web worker and node.

**Comment by Pessimistress** [comment]
> it just avoids the crash but since the params are not supported, the returned image is not what was expected.

I will drop the options for now since they are not universally supported.

**Comment by Pessimistress** [comment]
> It might be possible to perform these operations on the returned data in Javascript by rendering it into ImageData (via an invisible canvas)

`ImageBitmap`'s primary use case is in a web worker, where Canvas is not available. If `texImage2D` can only handle one orientation of `ImageBitmap`, then we have to flip the HTML images instead.
